docs/deployment/add-ons/cloudwatch/README.md

```diff
@@ -6,9 +6,9 @@ The EKS Cluster will need a IAM role asscoiated with one of its worker nodes wit
 export CLUSTER_NAME=<>
 export CLUSTER_REGION=<>
 export NODEGROUP_NAME=$(aws eks list-nodegroups --cluster-name $CLUSTER_NAME --region $CLUSTER_REGION | jq -r '.nodegroups[0]')
```

Comment:
what about other nodegroups?

Reply:
https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-prerequisites.html says to choose one of your worker nodes. Afaik the eksctl create command we ask users to use only ends up having one nodegroup